contracts/src/integrations/cert/EigenDACertVerifier.sol

```diff
@@ -34,7 +40,7 @@ contract EigenDACertVerifier is
     bytes internal _quorumNumbersRequired;
 
     uint8 internal constant MAJOR_VERSION = 3;
-    uint8 internal constant MINOR_VERSION = 0;
+    uint8 internal constant MINOR_VERSION = 1;
```

Comment:
are we gonna have a github release correspond to this semver bump?

Reply:
Hmm maybe we should huh. Let's discuss this with rest of team.